docs/1.34/faq/migrate-to-self-hosted-fq15.mdx

```diff
@@ -4,7 +4,7 @@ export const meta = {
   articleGroup: 'Prisma 1 Cloud Sunset',
 }
 
-# Migrating Away from Prisma 1 Cloud
+## FAQ
```

Comment:
I removed the duplicate title and I _believe_ that by making this an H2 it will apply the typical FAQ styles.